doc/changelog/07-vernac-commands-and-options/15690-micromega-deprecate-Simplex-option.rst

```diff
@@ -0,0 +1,4 @@
+- **Removed:**
+  `Simplex` flag, that was deprecated in 8.13
```

Comment:
The documentation says 8.14. Is 8.13 correct here? Also, you could remind here that it was the default behavior anyway.

Reply:
You are right, this is 8.14. I'll update.